app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,16 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self, nickname: str, favourite_dish: str,
+                 skill_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def player_info(self) -> str:
+        nk = self.nickname
+        sk = self._skill_level
```

Comment:
no need to create variable if it used only once

Reply:
How to fix flake8 test without it?

Reply:
E501 line too long (82 > 79 characters)